src/darker/git.py

```diff
@@ -26,8 +27,24 @@ def should_reformat_file(path: Path) -> bool:
     return path.exists() and path.suffix == ".py"
 
 
-def git_diff_name_only(paths: Iterable[Path], cwd: Path) -> Set[Path]:
-    """Run ``git diff --name-only`` and return file names from the output
+def _git_check_output_lines(cmd: List[str], cwd: Path) -> List[str]:
+    """Log command line, run Git, split stdout to lines, exit with 123 on error"""
+    logger.debug("[%s]$ %s", cwd, " ".join(cmd))
+    try:
+        return check_output(cmd, cwd=str(cwd)).decode("utf-8").splitlines()
+    except CalledProcessError as exc_info:
+        if exc_info.returncode == 128:
+            # Bad revision or another Git failure
+            sys.exit(123)
```

Comment:
123 or 128 ? maybe you tried to keep the git error code ?

Reply:
123 is what Black returns. I thought it's good to stay as similar as possible to it for the benefit of integration with other tools.

Reply:
From [Black's README]:
```
  --check                         Don't write the files back, just return the
                                  status.  Return code 0 means nothing would
                                  change.  Return code 1 means some files
                                  would be reformatted. Return code 123 means
                                  there was an internal error.
```